Reaqtor/Samples/Shebang/Reaqtor.Shebang.Client/QueryOperators.Generated.cs

```diff
@@ -295,31 +295,31 @@ public static IAsyncReactiveQbservable<IList<TSource>> Buffer<TSource>(this IAsy
             return source.Provider.CreateQbservable<IList<TSource>>(Expression.Call(((MethodInfo)MethodBase.GetCurrentMethod()).MakeGenericMethod(typeof(TSource)), source.Expression, Expression.Constant(count, typeof(int))));
         }
 
-        [KnownResource("rx://observable/buffer/duration/shift")]
-        public static IAsyncReactiveQbservable<IList<TSource>> Buffer<TSource>(this IAsyncReactiveQbservable<TSource> source, TimeSpan duration, TimeSpan shift)
+        [KnownResource("rx://observable/buffer/duration/count")]
```

Comment:
Unrelated to your change, but we can put a little more OrderBy/ThenBy clauses in the T4 file where we enumerate over the methods on Subscribable, such that regeneration of the code is more stable and doesn't bring in redundant reordering changes. E.g. we can order by name, then by generic parameter count,, then by a string.Join of the generic parameter names, then by parameter count,  and finally by a string.Join of the parameter names and their types.

This can of course be done in a separate PR. Both T4 files for client and service have to be updated, or we can refactor the common bits in a T4 file that gets included by both. There are some examples on how we've done that for other things, e.g. IrpEventSource.tt.